src/Controller/Activity/ActivityController.php

```diff
@@ -40,7 +40,7 @@ public function indexAction()
             $groups = $em->getRepository(Group::class)->findAllFor($user);
         }
 
-        $activities = $em->getRepository(Activity::class)->findUpcomingByGroup($groups);
+        $activities = $em->getRepository(Activity::class)->findUpcomingByGroupWithoutHidden($groups);
```

Comment:
seems like a really verbose function name, @jasperweyne thoughts? maybe findUpcomingVisibleByGroup, since you'll want only the visible ones and then sort on group. Seems like a more natural naming idea

Reply:
Agreed